### PR TITLE
Added new HDSC targets HC32F451 and HC32F452. Changed PN from hc32m12…

### PR DESCRIPTION
**Pull request by lennvn**
…0 to hc32m120x6.

### CODE REVIEW

File: pyocd/target/builtin/__init__.py [hunk 1]
```diff
@@ -242,14 +243,18 @@
...
+          'hc32f452xe' : target_HC32F45x.HC32F452xE,
           'hc32f460xc' : target_HC32F460.HC32F460xC,
           'hc32f460xe' : target_HC32F460.HC32F460xE,
           'hc32f4a0xg' : target_HC32F4A0.HC32F4A0xG,
           'hc32f4a0xi' : target_HC32F4A0.HC32F4A0xI,
           'hc32m423xa' : target_HC32M423.HC32M423xA,
           'hc32f120x6' : target_HC32x120.HC32F120x6TA,
           'hc32f120x8' : target_HC32x120.HC32F120x8TA,
-          'hc32m120' : target_HC32x120.HC32M120,
+          'hc32m120x6' : target_HC32x120.HC32M120,
```
Comment (flit) [inline pyocd/target/builtin/__init__.py:257]: Should `hc32m120` be deprecated but kept for backwards compatibility?

**Review by flit — CHANGES_REQUESTED** [review]
One question, one tiny change request.

**Comment by flit** [comment]
Oh, just noticed… can you please rebase and retarget this PR to the `develop` branch? (Since it adds new targets.) Thanks!

File: pyocd/target/builtin/__init__.py (hunk 1 above)
Comment (lennvn) [reply pyocd/target/builtin/__init__.py:257]: Thanks. I'll keep hc32m120.

**Comment by flit** [comment]
Thanks!

**Pull request merged**